ius-ecclesiae.csl

```diff
@@ -0,0 +1,683 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" initialize-with-hyphen="true" page-range-format="expanded">
+  <info>
+    <title>Ius Ecclesiae</title>
+    <title-short>IE</title-short>
+    <id>http://www.zotero.org/styles/ius-ecclesiae</id>
+    <link href="http://www.zotero.org/styles/ius-ecclesiae" rel="self"/>
```

Comment:
Can you add a "template" link? (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)